src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -314,7 +314,7 @@ static ucs_status_t uct_rdamcm_cm_ep_server_init(uct_rdmacm_cm_ep_t *cep,
     ucs_status_t           status;
     char                   ep_str[UCT_RDMACM_EP_STRING_LEN];
 
-    cep->id     = event->id;
+    cep->id = event->id;
```

Comment:
align =

Reply:
Need probably to update clang-format rules: 
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=12132&view=logs&jobId=cc064a77-22b5-56bf-ecc0-70b5fe764261&j=cc064a77-22b5-56bf-ecc0-70b5fe764261&t=aedfd754-44a6-53e6-6843-8659d800fee2

```
-    cep->id     = event->id;
+    cep->id = event->id;
```